src/main/java/com/mraof/minestuck/client/renderer/entity/UnderlingRenderer.java

```diff
@@ -64,7 +63,7 @@ public ResourceLocation getTextureLocation(T entity)
 		{
 			try
 			{
-				IResourceManager resourceManager = Minecraft.getInstance().getResourceManager();
+				ResourceManager resourceManager = Minecraft.getInstance().getResourceManager();
 				NativeImage base = SimpleTexture.TextureData.load(resourceManager, super.getTextureLocation(entity)).getImage();
 				NativeImage texture = SimpleTexture.TextureData.load(resourceManager, getGristTexture(entity)).getImage();
```

Comment:
Many components of SimpleTexture have become private or protected. Im not sure if we will have to create our own class to extend AbstractTexture the way Geckolib does with GeoAbstractTexture

Reply:
Fixed by Amiwyn